### DISCUSSION TITLE
Ender 3 S1 Extruder problem

### DISCUSSION THREAD

**Discussion by KarelVDM** [mriscoc/Ender3V2S1#1369] [Q&A]
So I have an Ender 3 V3 SE and recently aqquired an Ender 3 S1. The S1 lacked 2 features I got comfortable with on the V3SE

- Visible Bed Level 
- Filament Load/Unload. 

So after searching I came across this Custom Firmware.  My Ender 3 S1 has the STM32F103 chip so I downloaded the Firmware for that. 

After loading the Firmware, I rebooted the Printer. 
-  Visible Bed leveling now available and was able to "Fine Tune" my bed level. 
-  Filament Load and Unload options work perfectly. 

Here is the problem.... When I want to print something, the extruder doesnt move at all..... Thus its not feeding the filament to print....  

Did I miss a setting of calibration along the way?  

TIA 
Karel

**Comment by mriscoc** [comment]
Check your extruder minimum temperature, that is a security feature and the printer will not extrude filament is the current hotend temperature is under that value.

**Reply by KarelVDM** [reply]
Will check that tomorrow. Thanx. 
Flashed back OEM Creality software and currently running a print on it.

**Reply by KarelVDM** [reply]
> Check your extruder minimum temperature, that is a security feature and the printer will not extrude filament is the current hotend temperature is under that value.

Thanx This was the problem... 

I have a different problem now....   for some reason the runout sensor is triggered every 3-6 layers.....

**Reply by mriscoc** [reply]
This firmware includes support for runout sensors. If your filament is being ejected for no apparent reason, please check the wiring or settings of your filament run out sensor https://github.com/mriscoc/Ender3V2S1/wiki/Filament-Settings-Menu#run-out-sensor
